test/functional/provider/admin/account/users_controller_test.rb

```diff
@@ -13,7 +13,7 @@ def setup
   def test_update_blank_member_permission_ids
     assert_equal @default_ids, @user.admin_sections.to_a
 
-    put provider_admin_account_user_path(@user.id), params: {user: { member_permission_ids: [] }}
+    put provider_admin_account_user_path(@user.id), params: {user: { member_permission_ids: [''] }}
```

Comment:
`['']` is the param that is sent from the UI for `member_permission_ids` when it is empty.
![image](https://user-images.githubusercontent.com/11318903/73198733-d91e4a80-4133-11ea-978d-ab0290fe4b84.png)

In the original code, the one currently in master, before this PR, this is what we are testing (functional test):
https://github.com/3scale/porta/blob/837cbb815afda6f881c0fd497701725a96502973/test/functional/provider/admin/account/users_controller_test.rb#L16
Which sends this as params:
![image](https://user-images.githubusercontent.com/11318903/73198866-197dc880-4134-11ea-983a-9b2cc19f4201.png)

However, when done as an Integration test, the line `put provider_admin_account_user_path(@user.id), params: {user: { member_permission_ids: [] }}` does not work. Because the param `member_permission_ids` is not sent at all if it is done this way.

However, the line `put provider_admin_account_user_path(@user.id), params: {user: { member_permission_ids: [''] }}` works well.

This is independent of having or not this file: https://github.com/3scale/porta/pull/1583/commits/6ce265a6a05cd27690ddd806effd130baffc1d2c#diff-4b36adfa3f694588ae61b0d05c5ac580R1